Src/Base/AMReX_BLBackTrace.cpp

```diff
@@ -220,8 +220,14 @@ BLBackTrace::print_backtrace_info (FILE* f)
                     const std::string full_cmd = cmd + " " + print_buff;
                     addr2line_result = run_command(full_cmd);
                 }
-            } else if (amrex::system::call_addr2line && have_addr2line &&
-                       !amrex::system::exename.empty())
+
+                if (addr2line_result.find('?') != std::string::npos) {
+                    addr2line_result.clear();
+                }
+            }
+
+            if ((!(amrex::system::call_addr2line && have_eu_addr2line) || addr2line_result.empty()) &&
+                amrex::system::call_addr2line && have_addr2line && !amrex::system::exename.empty())
```

Comment:
```suggestion
            if (addr2line_result.empty() &&
                amrex::system::call_addr2line && have_addr2line && !amrex::system::exename.empty())
```
I think it can be simplified.